docs/section22/Rule22-23.md

```diff
@@ -0,0 +1,50 @@
+
+# CHW&CW - Rule 22-23  
+
+**Schema Version:** 0.0.10 **Mandatory Rule:** True
+**Rule ID:** 22-23  
+**Rule Description:** The baseline chiller efficiencies shall be modeled at the minimum efficiency levels, both part load and full load, in accordance with Tables G3.5.3.  
```

Comment:
I suggest that we split this RDS to separately verify full load and part load efficiency. All tools can output the full load, but some may not be able to do part load.